index.html

```diff
@@ -24,7 +24,7 @@ <h1 class="text_header">The best cakes in town <br>deliverd to your door</h1>
                 </div>
             </div>
         </header>
-        <nav class="nav_mun box" style="background-color: #e3f2fd;">
```

Comment:
Try to avaoid inline styling and only use css files to style, to avoid confusion plus is not really useful to use inline stylying, I am talking about the ``` <nav class="nav_mun box" style="background-color: #e3f2fd;">```